### DISCUSSION TITLE
Timeout Issue with Charts on Athena Data Source

### DISCUSSION THREAD

**Discussion by novdanody** [apache/superset#23177] [Q&A / Help]
Hi, 

I am querying AWS Athena on Chart with GLOBAL_ASYNC_QUERY=True, and I can see that the query failed but it seems like Superset Chart UI still tries to load the result. I tried changing the timeout on SUPERSET_WEBSERVER_TIMEOUT, SQLLAB_ASYNC_TIME_LIMIT_SEC to 10 seconds but the UI still shows loading. The gunicorn timeout is also 60s by default, but the UI still loads even after 10 minutes. Am I missing a flag that could force chart timeout after 10 seconds? Alternatively, is there a way to make superset not load chart by default upon opening a chart, but requires user to press "update chart" instead?

**Comment by LeoDiep** [comment]
Hi, as I know, chart will load data from cached data first by default. If there is no cached data then chart will start query and calculate again.

Sometime I encouter the same problem. After waiting for about 1,5 mins I know that I should hit Refresh the web browser and now the chart will show the data
